canonical-web-and-design/ubuntu.com
# Add usabilla embed code to advantage

## Done

Added Usabilla embed code to /advantage base template

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that a feedback widget is visible when you've been on the page for at least 20 seconds.


## Issue / Card

Fixes #7559
